removed use of installation tab (#1778)
keycloak/keycloak-documentation: server_admin/topics/clients/oidc/con-audience.adoc

.Audience protocol mapper
image:images/audience_mapper.png[]

> * From the <<_client_installation, Client details tab>> of the *good-service* client, you can generate the adapter configuration and you can confirm that _verify-token-audience_ option will be set to *true*. This forces the adapter to verify the audience if you use this configuration.

Reviewer: ```suggestion
* From the <<_client_installation, Client details tab>> of the *good-service* client, you can generate the adapter configuration and confirm that _verify-token-audience_  is set to *true*. This action forces the adapter to verify the audience if you use this configuration.
```
